src/app/layout.tsx

```diff
@@ -31,7 +31,7 @@ const RootLayout: React.FC<PropsWithChildren> = ({ children }) => {
             }
           >
             {children}
-            {`test ${client.provider._network.chainId}`}
+            {`Chain ID: ${client.provider._network.chainId}`}
```

Comment:
I'm not sure if this will work, due to the app folder running on the server by default.

I would, for now, stick to the `pages` folder instead of the experimental `app` folder. You can take [Byont-Ventures/web3-template](https://github.com/Byont-Ventures/web3-template/tree/main) as inspiration.

Reply:
Done 👍 Also removed the chain ID (for now). Refactored documentation in a way that I think makes more sense. Chain ID explanations etc could come later down the line when they might want to add a new network to their wallet or something